server/netty-server/loom/src/main/scala/sttp/tapir/server/netty/loom/internal/ox/OxDispatcher.scala

```diff
@@ -0,0 +1,18 @@
+package sttp.tapir.server.netty.loom.internal.ox
+
+import ox.*
+import ox.channels.Source
+import ox.channels.Actor
+
+private[loom] class OxDispatcher(using ox: Ox) {
```

Comment:
maybe we can pass the capability in a secondary constructor so that it's not captured as a field?